next.config.js

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable import/no-extraneous-dependencies */
+/* eslint-disable no-unused-vars */
+const path = require('path');
+const glob = require('glob');
+
+module.exports = {
+    // eslint-disable-next-line no-unused-vars
+    webpack: (config, { dev }) => {
```

Comment:
why are you destructuring `dev` if you are not using it. remove the eslint disable on line 2 and 7

Reply:
Resolved